README.md

```diff
@@ -176,8 +176,8 @@ boolean authenticity = hmacValidator.validateHMAC(hmacKey, signKey, payload);
 ~~~~
 If this bool returns true, one can proceed to deserialize against the desired webhook type:
 ~~~~ java
-BankingWebhookHandler webhookHandler = new BankingWebhookHandler();
-AccountHolderNotificationRequest accountHolderNotificationRequest = webhookHandler.getAccountHolderNotificationRequest(payload);
+BankingWebhookHandler webhookHandler = new BankingWebhookHandler(payload);
+AccountHolderNotificationRequest accountHolderNotificationRequest = webhookHandler.getAccountHolderNotificationRequest().get();
```

Comment:
The `Optional.get` method is used when you are sure about the `payload` contents. Normally that won't be the case, so a user can attach a callback for each case and only the valid one will be invoked:  

```suggestion
// onAccountHolderNotificationRequest
webhookHandler.getAccountHolderNotificationRequest().ifPresent((AccountHolderNotificationRequest event) -> {
    System.out.println(event.getData().getBalancePlatform());
});
// onBalanceAccountNotificationRequest
webhookHandler.getBalanceAccountNotificationRequest().ifPresent((BalanceAccountNotificationRequest event) -> {
    System.out.println(event.getData().getBalanceAccount());
});

```